apps/files/src/components/FileUpload.vue

```diff
@@ -52,7 +52,7 @@ export default {
           this.upload(file)
         } else {
           this.showNotification({
-            title: `Upload for ${file.name} failed - File allready exists`,
+            title: `${this.$gettext('Upload for')} ${file.name} ${this.$gettext('failed - File allready exists')}`,
             type: 'error'
```

Comment:
Interpolation required. Position of the variable in the string is different depending on the language